### ISSUE TITLE
[Cosmetic][Quest][Ammen Vale] The Emmiter event missing

### ISSUE THREAD

**Issue by ashphoenix92** [dalaranwow/dalaran-wow#5516]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Upone completeting the quest no scripted event happens.

**Expected behaviour**: <The image of Technician Dyvuun appears and turns to Zhanaa.>
Image of Technician Dyvuun says: By the seven Ata'mal crystals! Others have survived the crash! Zhanaa, is that you? It is so good to see you again!
Dyvuun turns to you.
Image of Technician Dyvuun says: And who do we have here? No doubt th... crzzzk ...ade this communication possible? I give you greetings fro... psshzzzk... Azure Watch.
Image of Technician Dyvuun says: Judging by your signal we put you... ust southeast of the larger part of the island we crashed onto. Cross the Crystalbrook River to ge... kshhhhk zzzt. Sorry, there's a lot of stat...
The image fades.
Technician Zhanaa says: This is great news! Your efforts have indeed saved us, <name>!
**Steps to reproduce the problem**:

1 None. Just quest complete the quest as regular and event does seem to be unscripted

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

https://wow.gamepedia.com/Quest:The_Emitter

http://www.wowhead.com/quest=9312/the-emitter

**Include Screenshots from the issue if necessary**
